cmds/exp/tcz/tcz.go

```diff
@@ -118,9 +118,8 @@ func clonetree(tree string) error {
 		}
 
 		return nil
-	})
-	if err != nil {
-		l.Fatalf("Clone tree: %v", err)
+	}); err != nil {
```

Comment:
in this case, you use the err := form; in other cases, you change it predeclare the error, and I don't see the point.